### ISSUE TITLE
Gripper Toggle

### ISSUE THREAD

**Issue by adamsong** [dwpage/frc2015#1]
The manage gripper button appears seems like it will not work. Because you are reading the current value of the solenoid to invert it when you set it to off it would clear that and then the code would not know the state of the solenoid.

**Comment by dwpage** [comment]
Good call, I wrote the original one as a single solenoid and when I updated it, didn't think about it. How would you fix it?

**Comment by adamsong** [comment]
I would not bother to turn off the solenoid, just leave it in the same state. If, however there is a reason to turn it off that I am not aware of, then I would add a boolean that holds the state of the solenoid.

**Comment by dwpage** [comment]
In general it would be so you don't continually keep it pressurized, but it isn't really necessary. But let's say it was, how would you fix it.

**Comment by adamsong** [comment]
Boolean variable
